core/src/SYCL/Kokkos_SYCL_Parallel_Reduce.hpp

```diff
@@ -441,9 +433,69 @@ class ParallelReduce<FunctorType, Kokkos::RangePolicy<Traits...>, ReducerType,
                       std::min(n_wgroups, wgroup_size));
                 }
               }
-            });
+            };
+            return lambda;
+          };
+
+      auto parallel_reduce_event = q.submit([&](sycl::handler& cgh) {
+        sycl::accessor<unsigned int, 1, sycl::access::mode::read_write,
+                       sycl::access::target::local>
+            num_teams_done(1, cgh);
+
+        auto dummy_reduction_lambda =
```

Comment:
Why is called `dummy`?

Reply:
Because it's base dummy memory pointers.